spec/meilisearch/index/search/facets_distribution_spec.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+RSpec.describe 'MeiliSearch::Index - Search' do
+  before(:all) do
+    @documents = [
+      { objectId: 123,  title: 'Pride and Prejudice',                    genre: 'romance' },
+      { objectId: 456,  title: 'Le Petit Prince',                        genre: 'adventure' },
+      { objectId: 1,    title: 'Alice In Wonderland',                    genre: 'adventure' },
+      { objectId: 2,    title: 'Le Rouge et le Noir',                    genre: 'romance' },
+      { objectId: 1344, title: 'The Hobbit',                             genre: 'adventure' },
+      { objectId: 4,    title: 'Harry Potter and the Half-Blood Prince', genre: 'fantasy' },
+      { objectId: 42,   title: 'The Hitchhiker\'s Guide to the Galaxy' }
+    ]
```

Comment:
Can you update the `@documents` to make another test with a `facetsDistribution` containing more than one field?